ui/src/app/lib/atlasmap-data-mapper/components/document-definition.component.ts

```diff
@@ -61,6 +60,14 @@ export class DocumentDefinitionComponent implements OnInit {
     this.documents = this.cfg.getDocs(this.isSource);
   }
 
+  getLineMachine(): any {
+      return this.lineMachine;
+  }
+
+  setLineMachine(lm: any): void {
+      this.lineMachine = lm;
+  }
```

Comment:
I’d reformat this to two spaces indenting to stay consistent with the rest of the file